.taprc

```diff
@@ -3,3 +3,7 @@ jsx: false
 flow: false
 check-coverage: false
 coverage: true
```

Comment:
I think all these options can removed as they have sensible defaults

Reply:
Only the threshold configs are present now.